monitor/docker-compose.yml

```diff
@@ -10,5 +10,7 @@ services:
     env_file: ./.env
     environment: 
       - NODE_ENV=production
+    volumes:
+      - ./responses:/mono/monitor/responses
```

Comment:
Could you double check this? I thought that relative paths cannot be used for the docker volume

Reply:
I'll double check, but so far I didn't see any errors. Documentation of docker-compose [mentions relative paths are allowed](https://docs.docker.com/compose/compose-file/compose-file-v2/#volumes)

Reply:
Thanks for the link! The composer is smarter than pure `docker`! No additional check is required then.